platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -10,7 +10,7 @@
 <meta name="twitter:description" content="TBD">
 <meta name="twitter:image" content="https://gradle.org/images/releases/gradle-8.13.png">
```

Comment:
💅 While we are here, could we replace some/all of the version numbers here as well?